frugalos_segment/src/config.rs

```diff
@@ -72,6 +73,7 @@ impl ClusterConfig {
     }
 }
 
+/// A set of `ClusterMember` which MAY have a replica of original data.
```

Comment:
`ClusterMember` → `ClusterMember`s ??

If this module private struct instances are created only by `ClusterConfig::candidates`, I'd like to mention it.

Reply:
`ClusterMember`s is correct and added your suggested comment.